Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm

```diff
@@ -52,17 +54,28 @@ static Class interactionRegionLayerClass()
     static NeverDestroyed<RetainPtr<NSDictionary>> interactionRegionEffectUserInfo;
     static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
-        RCPRemoteEffectInputTypes allowedInputs = RCPRemoteEffectInputTypesAll ^ RCPRemoteEffectInputTypePointer;
-        interactionRegionEffectUserInfo.get() = @{ RCPAllowedInputTypesUserInfoKey: @(allowedInputs) };
+        interactionRegionEffectUserInfo.get() = @{ RCPAllowedInputTypesUserInfoKey: @(interactionRegionInputTypes) };
     });
     return interactionRegionEffectUserInfo.get().get();
 }
 
+static float brightnessMultiplier()
+{
+    static float multiplier = -1;
+    if (multiplier < 0) {
```

Comment:
should this use the `dispatch_once` pattern instead?

Reply:
good point!
I was matching `scrollWheelMultiplier`, but dispatch_once is more readable!